packages/clerk-js/src/utils/__tests__/captcha.spec.ts

```
@@ -23,3 +23,229 @@
+
+      const { retrieveCaptchaInfo } = await import('../captcha/retrieveCaptchaInfo');
+      const result = retrieveCaptchaInfo(mockClerk as any);
+
+      expect(result.nonce).toBeUndefined();
+    });
+  });
+
+  describe('CaptchaOptions type support', () => {
+    it('should accept nonce in CaptchaOptions type definition', () => {
+      // This test verifies that the CaptchaOptions type includes the nonce field
+      const validOptions: import('../captcha/types').CaptchaOptions = {
```

ESLint

[error] `import()` type annotations are forbidden.

(@typescript-eslint/consistent-type-imports)